src/connector.js

```diff
@@ -603,16 +613,6 @@ function initEngine() {
 			}
 		}
 
-		if ( urlParams.year ) {
-			const year = Number.parseInt( urlParams.year );
-			if ( Number.isInteger( year )  && ( year >= 2000 )  && ( year <= ( new Date().getFullYear() + 1 ) ) ) {
-				aqString += ' @uri=".ca/' + urlParams.year + '"';
-			}
-			else {
-				aqString += ' NOT @uri';
-			}
-		}
-
```

Comment:
I don't think you should be removing the year feature since it is used for Budget Canada.